gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/definition/Api.java

```diff
@@ -38,7 +38,7 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class Api extends ReactableApi<io.gravitee.definition.model.Api> {
+public class Api extends AbstractReactableApi<io.gravitee.definition.model.Api> {
```

Comment:
Why this change I can not understand ? 🙈

Reply:
I have extracted an interface from the abstract class to create my DebugApiReactable interface. Relying on interfaces loose the coupling compared to abstract class.